examples/pytorch/application/qwen_doc_search_QA_based_on_dashscope.ipynb

```diff
@@ -282,153 +300,386 @@
+      "  inflating: punkt/turkish.pickle    \n",
+      "  inflating: punkt/polish.pickle     \n",
+      "   creating: punkt/PY3/\n",
+      "  inflating: punkt/PY3/greek.pickle  \n",
+      "  inflating: punkt/PY3/estonian.pickle  \n",
+      "  inflating: punkt/PY3/turkish.pickle  \n",
+      "  inflating: punkt/PY3/polish.pickle  \n",
+      "  inflating: punkt/PY3/russian.pickle  \n",
+      "  inflating: punkt/PY3/czech.pickle  \n",
+      "  inflating: punkt/PY3/portuguese.pickle  \n",
+      "  inflating: punkt/PY3/README        \n",
+      "  inflating: punkt/PY3/dutch.pickle  \n",
+      "  inflating: punkt/PY3/norwegian.pickle  \n",
+      "  inflating: punkt/PY3/slovene.pickle  \n",
+      "  inflating: punkt/PY3/english.pickle  \n",
+      "  inflating: punkt/PY3/danish.pickle  \n",
+      "  inflating: punkt/PY3/finnish.pickle  \n",
+      "  inflating: punkt/PY3/swedish.pickle  \n",
+      "  inflating: punkt/PY3/spanish.pickle  \n",
+      "  inflating: punkt/PY3/german.pickle  \n",
+      "  inflating: punkt/PY3/italian.pickle  \n",
+      "  inflating: punkt/PY3/french.pickle  \n",
+      "  inflating: punkt/russian.pickle    \n",
+      "  inflating: punkt/czech.pickle      \n",
+      "  inflating: punkt/portuguese.pickle  \n",
+      "  inflating: punkt/README            \n",
+      "  inflating: punkt/dutch.pickle      \n",
+      "  inflating: punkt/norwegian.pickle  \n",
+      "  inflating: punkt/slovene.pickle    \n",
+      "  inflating: punkt/english.pickle    \n",
+      "  inflating: punkt/danish.pickle     \n",
+      "  inflating: punkt/finnish.pickle    \n",
+      "  inflating: punkt/swedish.pickle    \n",
+      "  inflating: punkt/spanish.pickle    \n",
+      "  inflating: punkt/german.pickle     \n",
+      "  inflating: punkt/italian.pickle    \n",
+      "  inflating: punkt/french.pickle     \n",
+      "  inflating: punkt/.DS_Store         \n",
+      "  inflating: punkt/PY3/malayalam.pickle  \n",
+      "  inflating: punkt/malayalam.pickle  \n",
+      "Archive:  averaged_perceptron_tagger.zip\n",
+      "   creating: averaged_perceptron_tagger/\n",
+      "  inflating: averaged_perceptron_tagger/averaged_perceptron_tagger.pickle  \n"
      ]
     }
    ],
    "source": [
-    "# query the top 1 results\n",
-    "question = '清华博士发生了什么？'\n",
-    "context = search_relevant_context(question, topk=1)\n",
-    "print(context)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "730abebb-1f5a-4fb9-b035-fb2ae09a31c9",
-   "metadata": {
-    "ExecutionIndicator": {
-     "show": true
-    },
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# initialize qwen 7B model\n",
-    "from modelscope import AutoModelForCausalLM, AutoTokenizer\n",
-    "from modelscope import GenerationConfig\n",
+    "!mkdir -p /root/nltk_data/tokenizers\n",
+    "!mkdir -p /root/nltk_data/taggers\n",
+    "!cp /mnt/workspace/punkt.zip /root/nltk_data/tokenizers\n",
```

Comment:
it is better to provide the url of these files and execute `wget` to download to make sure this cell can run successfully